File: 1-9/1/Main.java
import java.util.stream.IntStream;

public class Main {
    public static void main(String[] args) {
        int sum = IntStream.range(1, 1000)
                .filter(i -> i % 3 == 0 || i % 5 == 0)
                .sum();
        System.out.println(sum);
    }
}

File: 1-9/1/main.cpp
#include <iostream>

int main() {
    int sum = 0;

    for (int i = 0; i < 1000; i++)
        sum = (i % 3 == 0 || i % 5 == 0) ? sum + i : sum;

    std::cout << "Sum: " << sum << std::endl;

    return 0;
}